Wishlist/wishlist.css

```diff
@@ -0,0 +1,180 @@
+*{
+    box-sizing: border-box;
+    margin: 0;
+    padding: 0;
+}
+
+body{
+   font-family: 'Times New Roman', Times, serif;
+   font-size: 100%;
+    height: 100%;
+}
+
+.container{
+    width: 100%;
+    height: auto;
+    border: solid;
+    display: flex;
+    flex-direction: column;
+    align-items: center;
+    font-family:Arial, Helvetica, sans-serif ;
+    background-image: linear-gradient(black, purple);
+}
+.header{
+    height: 7vh;
+    width: 80%;
+    display: flex;
+    background-color: #000000;
+    color: white;
+    flex-direction: row;
+    border-radius: 4px;
+    justify-content: center;
+    align-items: center;
+}
+.header .header-nav-items{
+    display: flex;
+    justify-content: space-between;
+    align-items: center;
+    
+    margin-right: auto;
+    width: 30%;
+}
+.header-nav-items .nav-item{
+    color: white;
+    font-size: 1.1rem;
+    padding: 0.5rem;
+    transition: 100ms;
+}
+.header-nav-items .nav-item:hover{
+    border: 1px solid white;
+}
+.header-nav-items .nav-item a{ 
+    all: unset;
+}
+
+.header .header-title{
+    width: 40%;
+    text-align: center;
+    font-size: 2rem;
+    font-weight: 1000;
+}
+.header .header-title span{
+    background-color: #f3ec78;
```

Comment:
using color variables is a good practice instead of direct applying color code. You can change use root variable instead